core/src/main/java/com/radixdlt/RadixNodeModule.java

```diff
@@ -264,7 +264,8 @@ protected void configure() {
     var databasePath = properties.get("db.location", ".//RADIXDB");
     var mempoolMaxSize = properties.get("mempool.maxSize", 50);
     var mempoolConfig = new RustMempoolConfig(mempoolMaxSize);
-    var databaseConfig = new REv2DatabaseConfig.RocksDB(databasePath);
+    var enableAccountChangeIndex = properties.get("extension.account_change_index.enable", false);
```

Comment:
We'll want to include this on the `default.config.envsubst` - and turn it on for local development in the docker file - and we'll need to talk to Kim to ensure it's included in the full node set-up for a new release of the  Node CLI.

It might even be worth defaulting to `true` to start with.

Reply:
It's added to `default.config.envsubst` (and `Dockerfile.core`) in this PR already.

Should I make it default to `true` everywhere?

Reply:
Ah sorry I missed that! I think let's default it to true for now, and we can consider a more clear Validator / Full Node distinction for RCnet-V2